config/local.yaml

```diff
@@ -295,7 +295,7 @@ checks:
     type: smoke
   ci-load:
     options:
-      content-size: 50000
+      file-sizes: [31457280, 104857600, 314572800] # [30MB, 100MB, 300MB]  
```

Comment:
For the local.yaml, keep some smaller sizes, maybe max to 3 mb.
Local deployments do not have persistant-storage supported.